### ISSUE TITLE
[SC002] validate layout and description task 5

### ISSUE THREAD

**Issue by vuthypouk** [pnc-virtual-company/2018-selection#19]
please read and understand about the interface with the requirement  some point that you want to change

**Comment by CheaChan** [comment]
I am already with some parts of this task, and I need more time to understand more and cleary.

**Comment by CheaChan** [comment]
I have already completed this tasks, but I am not clear on the part of "Student Information" selected or not selected with the global grade.

**Comment by dakhen-sok** [comment]
Hello Chan.As I see your comment I don't know that Do you agree with this layout or not.so please give us for some agree that you are agree to follow this layout.Thank you.

**Comment by nuon-neourng** [comment]
The layout is ok but it seem complexity.

**Comment by CheaChan** [comment]
Yes, Dakhen I agree with this and for the comment that I am not clear now, I understand more about this requirement.
Nuon yes it is really complexity.
